### ISSUE TITLE
boot2docker init error

### ISSUE THREAD

**Issue by alanyih** [boot2docker/boot2docker#1052]
boot2docker-1.8.1

Booting Core 6.3
Running Linux Kernel 4.0.9-boot2docker.
Checking boot options... Done.
...
/etc/init.d/tc-config: line 615: /etc/init.d/dhcp.sh: not found
Running boot2docker init script...
net.ipv4.ip_forward = 1
sysctl: cannot stat /proc/sys/net/ipv6/conf/all/forwarding: No such file or directory
sysctl: setting key "cannot stat %s": No such file or directory
sysctl: "cannot stat %s" is an unknown key
sysctl: setting key "cannot stat %s": No such file or directory
Segmentation fault
automount ...

[b2docker-181.png] (https://goo.gl/8Qyzuj)

**Comment by paolomainardi** [comment]
@alanyih are you on Windows ?

**Comment by cgudrian** [comment]
I'm facing the same problem and I'm on Windows 7.

**Comment by communityportal** [comment]
Same issue here. Ran docker many times on Oracle Virtualbox and created default machine many times successfully. After single try to recreate it with disk size of 30GB I've started to get that issue.

After machine startup, the 'docker' daemon is not working in such state (even in VM shell).

Tried different things with host-only adapter settings (with/without DHCP) but without success.
The machine only starts by manual docker-machine create command.

Running by Docker Quickstart when machine not exists causes:
```
Creating Machine default...
Running pre-create checks...
Creating machine...
Error creating machine: Error in driver during machine creation: Maximum number of retries (5) exceeded
Looks like something went wrong... Press any key to continue...
```

The Kitematic was able to create machine but end up in error (unfortunatelly I did not grabbed it), and clicking button to remove and set up machine again fixed the problem, But stil I need to create VM by myself (proxy setting in docker-machine create) so I cannot use machine created by Kitematic.

It looks like Kitematic creates Host-only adapter WITH DHCP enabled and this can be the fix.
But even when I create VM manually and select it to use Host-only adapter with DHCP enabled, the error still occurs and docker daemon don't start up.

To sum up:
- cannot even start quickstart console, it cannot create VM
- Kitematic can create boot2docker VM but I need proxy setting (I will try to add to /var/lib/boot2docker/profile maually)
- manually invoking docker-machine create can create a machine but error still occurs at startup and docker daemon is not running

**Comment by communityportal** [comment]
It looks like using Host-only adapter with DHCP enabled BREAKS starting docker daemon after second VM startup. Disabling DHCP solved the problem.

**Comment by skygragon** [comment]
I have the same error in boot2docker.log when I tried to use docker-machine to create a new vm in vsphere. The created vm has no ipv4 address assigned on eth0 thus failed to ssh.

@communityportal I have no idea about the "disabling DHCP" thing, could you help elaborate it more?
